internal/cli/opsmanager/logs/decrypt.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logs
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/decryption"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+type DecryptOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	cli.DownloaderOpts
+	inFileName                    string
+	kmipServerCAFileName          string
+	kmipClientCertificateFileName string
+	localKeyFileName              string
+}
+
+func (opts *DecryptOpts) initFiles() func() error {
+	// Validate the provided files can be open
+	return func() error {
+		return nil
+	}
+}
+
+func (opts *DecryptOpts) stdOutMode() bool {
+	return opts.Out == ""
+}
```

Comment:
[q] why do we have this function? Can’t you just use `opts.Out == ""` ?

Reply:
this was introduced in my commit, created it because we test it many times (3 to be precise)

Reply:
the decrypt command will print the results either to a file or to stdout (which is also the default mode if --out field is not set). This helper function should tell us what if the stdout mode is used (and depending on it different logic is used in 3 places).
IMO, the check should be like this
```
return opts.Out == ""

Reply:
I added a doc comment: `// stdOutMode returns true when the results should be printed to Stdout (--out|-o flag is not set)`